css/style.css

```diff
@@ -29,4 +29,12 @@ table {
 
 .box_cell:hover {
   background-color: #aaa;
-}
\ No newline at end of file
+}
+
+.red {
```

Comment:
I like that you added classes for this. I think that's far easier and more flexible than adding CSS styles to each element.